ScriptureRenderingPipeline/Renderers/TranslationWordsRenderer.cs

```diff
@@ -10,146 +10,148 @@
 using System.Threading.Tasks;
 using JsonSerializer = System.Text.Json.JsonSerializer;
 
 namespace ScriptureRenderingPipeline.Renderers
 {
-    public class TranslationWordsRenderer
-    {
-        public async Task RenderAsync(ZipFileSystem sourceDir, string basePath, string destinationDir, Template printTemplate, string repoUrl, string heading, ResourceContainer resourceContainer, string baseUrl, string userToRouteResourcesTo, string textDirection, string languageCode, string languageName, bool isBTTWriterProject = false)
-        {
-            var projectPath = resourceContainer.projects[0].path;
-            var categories = await LoadWordsAsync(sourceDir, sourceDir.Join(basePath, projectPath), baseUrl, userToRouteResourcesTo, languageCode);
-            var printBuilder = new StringBuilder();
-            var outputTasks = new List<Task>();
-            var outputIndex = new OutputIndex()
-            {
-                LanguageCode = languageCode,
-                LanguageName = languageName,
-                TextDirection = textDirection,
-                RepoUrl = repoUrl,
-                ResourceType = "tw",
-                ResourceTitle = heading,
-                Bible = null,
-                Words = new List<OutputWordCategory>()
-            };
-            foreach(var category in categories )
-            {
-                var outputCategory = new OutputWordCategory()
-                {
-                    Slug = category.Slug,
-                    Label = category.Title
-                };
-                var titleMapping = new Dictionary<string, string>(category.Words.Count);
-                var builder = new StringBuilder();
-                builder.AppendLine($"<h1>{category.Title}</h1>");
-                foreach(var word in category.Words)
-                {
-                    builder.AppendLine($"<div id=\"{word.Slug}\"></div>");
-                    builder.AppendLine(word.Content);
-                    builder.AppendLine("<hr/>");
-                    titleMapping.Add(word.Slug, word.Title.Trim());
-                    outputCategory.Words.Add(new OutputWord()
-                    {
-                        Slug = word.Slug,
-                        Label = word.Title
-                    });
-                }
-                
-                outputIndex.Words.Add(outputCategory);
+	public class TranslationWordsRenderer
+	{
+		public async Task RenderAsync(ZipFileSystem sourceDir, string basePath, string destinationDir, Template printTemplate, string repoUrl, string heading, ResourceContainer resourceContainer, string baseUrl, string userToRouteResourcesTo, string textDirection, string languageCode, string languageName, bool isBTTWriterProject = false, JsonElement appsMeta = new JsonElement())
+		{
+			var projectPath = resourceContainer.projects[0].path;
+			var categories = await LoadWordsAsync(sourceDir, sourceDir.Join(basePath, projectPath), baseUrl, userToRouteResourcesTo, languageCode);

```

Comment:
this line is really long and should be broken up to having a parameter on each line